test/mtln/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ add_library(
     "test_Paul.F90"
     "test_fhash.F90"
     "test_dispersive.F90"
+    # "test_wirePanel.F90"
```

Comment:
Is this correct? Please double check, it is easy to forget this is here and we may be assuming these are being tested.

Reply:
Is correct but will be removed soon. Some tests are in process of being moved to python wrapper as mtln standalone tests